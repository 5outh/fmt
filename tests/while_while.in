<?php
while(true){}
while(true){}
try{}catch(Exception $e){}
while(){}
for($i=0;$i<1;$i++){}
while(){}
while(){while(){}while(){}do{}while();}
?>