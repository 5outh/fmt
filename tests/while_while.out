<?php
do {
	$a = 0;
} while ();
while () {
	$a = 0;
}
while () {
	$a = 0;
}
?>